client/my-sites/patterns/components/grid-gallery/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { PatternsSection } from 'calypso/my-sites/patterns/components/section';
+import type { CSSProperties } from 'react';
+
+import './style.scss';
+
+type PatternsGridGalleryProps = {
+	title: string;
+	description: string;
+	itemMinWidth?: number;
+	list?: {
+		name?: string;
+		label?: string;
+		number: number;
+		image: string;
+		link: string;
+	}[];
+};
+
+export const PatternsGridGallery = ( {
+	title,
+	description,
+	itemMinWidth = 287,
+	list,
+}: PatternsGridGalleryProps ) => {
+	if ( ! list ) {
+		return null;
+	}
+
+	return (
+		<PatternsSection title={ title } description={ description }>
+			<div
+				className="patterns-grid-gallery"
+				style={ { '--item-min-width': `${ itemMinWidth }px` } as CSSProperties }
+			>
+				{ list.map( ( { name, label, number, image, link } ) => (
+					<a className="patterns-grid-gallery__item" href={ link } key={ name }>
```

Comment:
```suggestion
					<LocalizedLink className="patterns-grid-gallery__item" href={ link } key={ name }>
```
Also not critical for this PR, but we should use `LocalizedLink` here